src/components/Form/Form.vue

```diff
@@ -119,11 +119,33 @@ export default {
   }),
   created () {
     this.formValues = pipe(flatten, map(getNameOrLabel), valueToProp)(this.formFields)
+    this.addUpdateValueCallback()
   },
   mounted () {
     this.allControls = this.$refs.control
   },
+  watch: {
+    formFields: {
```

Comment:
perhaps u could be more precise here by filtering with a computed and watch this computed